packages/react-cosmos-redux-proxy/src/index.jsx

```diff
@@ -1,16 +1,25 @@
 import React from 'react';
-import _ from 'lodash';
+import omit from 'object.omit';
```

Comment:
I think that only webpack 2 has support for tree-shaking. On the other hand I guess you can use https://www.npmjs.com/package/lodash.omit.